oslabs-beta/KataLog
# Cf/sidebar header components

##Overview
Issue Type
[]Tech debt
Description

Feature: The header and sidebar have been moved to their own component to make them more modular. The header now contains a dropdown menu that allows the user to select what project they want to view logs for.

Bug: Pagination and `Chart` are now properly displaying logs

Steps to Reproduce Bug/Validate Feature/Confirm Tech Debt Fix

- From any page, click on the dropdown menu and select a `project`

- On the `Logs` page, click on the dropdown menu and select a `project`
- Select any page after page 1

- On the `Dashboard`, click on the dropdown menu and select a `project`
- Select a `hexagon` rendered from the `Chart` to view `logs` related to that `type`

Previous Behavior
Logs were not rendering on any pages past page 1

Expected Behavior
Pagination now functions as expected

How Has This Been Tested
See "Steps to Reproduce Bug/Validate Feature/Confirm Tech Debt Fix"

Screenshots & Videos
N/A

Additional Context
N/A
